contrib/k8s/dex-overlord.yaml

```diff
@@ -67,10 +81,18 @@ kind: ConfigMap
 metadata:
   name: dex-connectors
 data:
-  local-connector.json: |
+  connector.json: |
     [
       {
-        "type": "local",
-        "id": "local"
+        "id": "local",
+        "type": "local"
       }
+      # {
```

Comment:
Argh. Though that would be ignored by kubectl parsing the yaml -- didn't think about the fact it was a literal block and would make it into the configmap value.